data/json/monsters/fish.json

```
@@ -1306,11 +1306,11 @@
   {
     "id": "mon_fish_mackerel",
     "type": "MONSTER",
-    "copy-from": "mon_fish_medium",
-    "name": { "str_sp": "atlantic mackerel" },
+    "copy-from": "mon_fish_tiny",
+    "name": { "str_sp": "Atlantic mackerel" },
     "description": "An Atlantic mackerel, known for its distinctive striped pattern.",
-    "volume": "3000 ml",
-    "weight": "1600 g",
+    "volume": "850 ml",
+    "weight": "425 g",
```

```suggestion
    "weight": "425 g",
```
Googling Atlantic mackerel gives an average weight of 4.1 lbs or 1859.73 grams.

Then clicking through to NOAA fishery page provides the following. 
Biology
Atlantic mackerel grow fast, up to 16 ½ inches and 2.2 pounds. which would still be double the weight you've listed here. 997.903 grams.

It might be best to determine a source of truth and comment each fish where that source of truth was pulled from.